public/static/docs/api-reference/get_url.md

```diff
@@ -0,0 +1,84 @@
+# dvc.api.get_url()
+
+Return the URL
+([string](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)
+type)) to the storage location of a <abbr>data artifact</abbr> tracked by DVC.
+
+## Signature
+
+```py
```

Comment:
if we keep it one-line, probably no need for a separate section then

Reply:
Will make into a longer **Definition** section. Addressing in https://github.com/iterative/dvc.org/pull/908#pullrequestreview-356421226.